### PR TITLE
Update stop sign tutorial

### LINKED ISSUE #24831: Update Tutorial: Detect stop signs in images with Model Builder
Review to make sure it's up to date

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 63a4f3b7-a2a6-760a-534b-22788ba77b5c
* Version Independent ID: 1bee7901-164f-d9a7-9e7b-fd0ba55ad42e
* Content: [Tutorial: Detect objects in images with Model Builder - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder)
* Content Source: [docs/machine-learning/tutorials/object-detection-model-builder.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/tutorials/object-detection-model-builder.md)
* Product: **dotnet-ml**
* GitHub Login: @briacht
* Microsoft Alias: **brachtma**

### PR DESCRIPTION
**Pull request by jwood803**
Updates to the stop sign tutorial:
- Include VoTT instructions to get data file.
- Updates to new version of Model Builder

Fixes #24831 

[Preview](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder?branch=pr-en-us-25406)

### CODE REVIEW

**Comment by jwood803** [comment]
@luisquintanilla This should fix the object detection tutorial.

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 1]
```diff
@@ -160,41 +218,54 @@ The score shown on each detected bounding box indicates the **confidence** of th
...
 
-    ```bash
-    Predicted Boxes:
+     Invoke-RestMethod "https://localhost:<PORT>/predict" -Method Post -Body ($body | ConvertTo-Json) -ContentType "application/json"
+    ```
+
+1. If successful, the output should look similar to the text below:
 
-    Top: 89.453415, Left: 481.95343, Right: 724.8073, Bottom: 388.32385, Label: Stop-Sign, Score: 0.99539465
+    ```powershell
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/object-detection-model-builder.md:265]: Add some information on here explaining what the output is.

**Comment by luisquintanilla** [comment]
CLosing and reopening for build issues

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 2]
```diff
@@ -55,18 +56,76 @@ This sample creates a C# .NET Core console application that detects stop signs i
...
+
+1. Go back to the **Tags editor** (the second icon in the left toolbar shaped like a ribbon). In the top toolbar, select the **Export Project** icon (the last icon shaped like an arrow in a box), or use the keyboard shortcut <kbd>Ctrl</kbd>+<kbd>E</kbd>.
+
+    ![VoTT Export Button](../how-to-guides/media/vott/vott-export-button.png)
+
+This export will create a new folder called *vott-json-export* in your *Stop-Sign-Images* folder and will generate a JSON file named *StopSignObjDetection-export* in that new folder. You will use this JSON file in the next steps for training an object detection model in Model Builder.
 
 ## Create a console application
 
 1. In Visual Studio, create a **C# .NET Core console application** called *StopSignDetection*.
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/object-detection-model-builder.md:120]:
```suggestion
In Visual Studio, create a **C# .NET Core console application** called *StopSignDetection*.
```

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 3]
```diff
@@ -55,18 +56,76 @@ This sample creates a C# .NET Core console application that detects stop signs i
...
 
-## Choose a scenario
+## Create an `mbconfig` file
 
 1. In **Solution Explorer**, right-click the *StopSignDetection* project, and select **Add** > **Machine Learning** to open the Model Builder UI.
+1. In the dialog, name the Model Builder project **StopSignDetection**, and click **Add**.
+
+## Choose a scenario
+
 1. For this sample, the scenario is object detection. In the **Scenario** step of Model Builder, select the **Object Detection** scenario.
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/object-detection-model-builder.md:129]:
```suggestion
For this sample, the scenario is object detection. In the **Scenario** step of Model Builder, select the **Object Detection** scenario.
```

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 4]
```diff
@@ -160,43 +220,60 @@ The score shown on each detected bounding box indicates the **confidence** of th
...
-1. Open the *Program.cs* file in the *StopSignDetection* project, and add the following using statement at the top of the file to reference the *StopSignDetectionML.Model* project:
+### Console App
 
-    [!code-csharp [ProgramUsings](~/machinelearning-samples/samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/Program.cs#L2)]
+When adding a console app to your solution, you will be prompted to name the project.
 
-1. Download the following [test image](~/machinelearning-samples/samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/test-image1.jpeg).
-1. Create a new instance of the `ModelInput` class with the `ImageSource` property set to the filepath of your test image inside the `Main` method of your application. Replace the "Hello World" statement with the following code:
+1. Name the console project **StopSignDetection_Console**.
+1. Click *Add to solution** to add the project to your current solution.
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/object-detection-model-builder.md:237]:
```suggestion
1. Click **Add to solution** to add the project to your current solution.
```

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 5]
```diff
@@ -160,43 +220,60 @@ The score shown on each detected bounding box indicates the **confidence** of th
...
+    ```
 
-1. Print out the Prediction's output, including the label, coordinates, and score by adding the following code:
+### Web API
 
-    [!code-csharp [PrintResults](~/machinelearning-samples/samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/Program.cs#L17-L18)]
+When adding a web API to your solution, you will be prompted to name the project.
 
+1. Name the Web API project **StopSignDetection_API**.
+1. Click *Add to solution** to add the project to your current solution.
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/object-detection-model-builder.md:253]:
```suggestion
1. Click **Add to solution** to add the project to your current solution.
```

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ title: 'Tutorial: Detect objects in images with Model Builder'
 description: This tutorial illustrates how to build an object detection model using ML.NET Model Builder and Azure ML to detect stop signs in images.
 author: briacht
 ms.author: brachtma
-ms.date: 07/25/2021
+ms.date: 08/11/2021
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/object-detection-model-builder.md:6]:
```suggestion
ms.date: 08/21/2021
```

**Review by luisquintanilla — APPROVED** [review]
LGTM. Made some minor fixes. Thanks @jwood803

**Pull request merged**